site/2017/04/01/announcing-doctrine-4.rst

```diff
@@ -0,0 +1,97 @@
+both RDBMS and NoSQL databases at the same time!**
+
+Following latest trends
+-----------------------
+
+A really big thing we want to (re)introduce is an Active Record pattern. We've ran a poll on the #doctrine IRC
+channel and it turned out that 68% of developers are dying a little bit each time they are injecting a service which
+barely saves data to a database and miss simplicity of having an entity save itself.
+
+.. code-block:: php
+
+    use Doctrine;
+
+    /**
+     * @Doctrine\Entity(storage="MSSql")
+     * @Doctrine\ActiveRecord
+     */
+    class User
+    {
+        /** @Doctrine\Id */
+        public $id;
+
+        /** @Doctrine\Field(type="string") */
+        public $login;
+    }
+
+    $user = new User();
+    $user->load(10);
+    $user->login = 'malarzm';
+    $user->save();
+
+Thanks to ``@Doctrine\ActiveRecord`` annotation you're able to query for and save your entities easily. Please
+notice, that `User` class does not extend any internal Doctrine class - you are still decoupled from the ORM!
+
+We strongly believe that getting back to Active Record pattern is the way to go for us. We weren't able to
+fully get rid of old fashioned Data Mapper but you can expect its deprecation in one of first bug-fix releases
+and full removal shortly after.
+
+Another big step towards the highly expected Developer eXperience was initially painful, as it required many
+of us to come out of the Java bubble we live in, but we know it will be for the best. With the re-introduction of
+Active Record the obvious next step is making all Doctrine utilities available in an easy and sane way: please welcome
+facades AND short method names!
+
+.. code-block:: php
+
+    use Doctrine;
+    use Entity\MongoLog;
+    use Entity\User;
+
+    Doctrine::em()->start(); // start a transaction
+    $user = new User(); // this is stored in MSSql
+    $user = 'malarzm';
+    $user->save(); // not saved yet
+    $log = new MongoLog($user, 'was created'); // this will be stored in Mongo
+    $log->save(); // but not yet
+    Doctrine::em()->commit(); // commit a transaction ACROSS storages
+
+Big shout out goes to `Marco <https://twitter.com/Ocramius>`__, who initially had a heart attack when he
+heard the idea, but now after using the feature is a big proponent of facades. Be sure to watch for updates
+in all his libraries in the near future!
+
+Try it out now!
+---------------
+
+Uniting forces of all Doctrine developers has enabled us to ship an usable "alpha" version way sooner than originally
+anticipated. But the truly thrilling news is that thanks to tremendous help from guys with `3v4l.org <https://3v4l.org/>`__
+we have set up a sandbox environment so everybody can have a hands-on experience using the new version of Doctrine:
+please visit `3v4l.org/doctrine4 <https://ocrami.us/>`__ and share your thoughts in the comments section below!

```

Comment:
I propose that @Ocramius briefly swap the original video for https://www.youtube.com/watch?v=zVyrz8jnRw8 for more Symfony exposure.